### DISCUSSION TITLE
Strike as a BTC price option

### DISCUSSION THREAD

**Discussion by roberto188** [btcpayserver/btcpayserver#5824] [Ideas / Feature Requests]
Can you add Strike as a BTC rate option?Lots of people use Strike to pay, why not use their rate to generate invoices?

**Comment by lighthold** [comment]
Same request here:
I currently use BTCPayServer with the Strike.me Plugin, which works amazing, there is only one problem, the exchange rate for the invoice.
Because for German accounting rules i have the following issues. I need to get instantly FIAT because otherwise is will be taxed twice. Which means on the invoice if i say 1€, i also need to get 1€ and not 0.99ct because then i need to show the Gouvernement where that one cent was lost.
For EUR i cannot select the Strike Exchange rate and currently rely on Kraken (recommended) but since i get less sats than I need for the Strike exchange rate, i get exactly this 1ct difference.

https://docs.strike.me/api/issue-quote-for-invoice/

**Reply by pavlenex** [reply]
It's a reasonable request. @rockstardev since users use Strike as a wallet, they'd like to have Strike as a price source. Is it possible to do this, if so, perhaps we can log it as a good first issue, wdyt?

**Reply by lighthold** [reply]
Just a thought, could this potentially also be part of the Strike plugin itself?
